src/styles/tool-header.scss

```diff
@@ -160,7 +161,7 @@ $fd-tool-header-object-status-semantic-values: (
       }
     }
 
-    &-text {
+    .#{$block}__button-text {
```

Comment:
is the specificity needed here?

Reply:
Yea, at least till we refactor button component, `.fd-button.fd-button--emphasized` => `.fd-button--emphasized`